src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbstractClassNameCheck.java

```diff
@@ -64,8 +77,41 @@
  * <p>Example:</p>
  * <pre>
  * abstract class AbstractFirstClass {} // OK
- * abstract class SecondClass {} // violation, it should match the pattern "^Abstract.+$"
+ * abstract class SecondClass {} // violation, it should match pattern "^Abstract.+$"
  * class AbstractThirdClass {} // OK, no "abstract" modifier
+ * class FourthClass {} // OK
+ * </pre>
+ * <p>
+ * How to configure the {@code AbstractClassName} to
+ * check names with {@code ignoreName}:
```

Comment:
Please update to `How to configure the {@code AbstractClassName} to ignore name validation when class declared as 'abstract'`

Reply:
done